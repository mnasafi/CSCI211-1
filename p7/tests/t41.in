echo t41: an illegal command with arguments
yikes here are some bad arguments
echo end of test (to make sure you did not give up w/the illegal command)
