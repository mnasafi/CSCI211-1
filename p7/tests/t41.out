t41: an illegal command with arguments
end of test (to make sure you did not give up w/the illegal command)
